documentation/src/main/asciidoc/user_guide/server_protocols_hotrod.adoc

```diff
@@ -917,6 +917,38 @@ In order to avoid creating a TCP connection on each request (which is a costly o
 
 The number of connections per server, total number of connections, how long should a connection be kept idle in the pool before being closed - all these (and more) can be configured. Please refer to the javadoc of link:{javadocroot}/org/infinispan/client/hotrod/RemoteCacheManager.html[RemoteCacheManager] for a list of all possible configuration elements.
 
+==== Using internal/external IP addresses
+
+In some cases (e.g. OpenStack or Kubernetes with services exposed using load balancers) servers use different IP addresses for clustering and different for communicating with the client.
```

Comment:
Definitely not! This is something that depends on the environment very heavily. 

For example in Kubernetes and OpenStack there is no way to extract such mapping. This is why I came up with https://github.com/slaskawi/external-ip-proxy which exposes a REST service and allows it.